app/server.go

```diff
@@ -193,6 +196,22 @@ func NewServer(options ...Option) (*Server, error) {
 	// Use this app logger as the global logger (eventually remove all instances of global logging)
 	mlog.InitGlobalLogger(s.Log)
 
+	if *s.Config().LogSettings.EnableSentry {
+		if strings.Contains(SENTRY_DSN, "placeholder") {
+			mlog.Warn("Sentry reporting is enabled, but SENTRY_DSN is not set. Disabling reporting.")
+		} else {
+			if model.BuildHash == "dev" {
```

Comment:
If we're only injecting the DSN as part of the release process, we don't need this check. Or we could leave this check, hard code the DSN into the source and not have to do any injection in the release process. 0/5 on which we choose.

Reply:
Good point. I think it's cleaner to inject the DSN. I'll remove the check